scripts/update-dockerhub-description.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+set -euxf -o pipefail
+
+image_name="$1"
+readme_path="$2"
+
+dockerhub_repository="jaegertracing/$image_name"
+dockerhub_url="https://hub.docker.com/v2/repositories/$dockerhub_repository/"
+
+quay_repository="quay.io/jaegertracing/$image_name"
+quay_url="https://quay.io/api/v1/repository/$quay_repository/description"
+
+if [ ! -f "$readme_path" ]; then
+  echo "Warning: the dedicated README file for Docker image $image_name was not found at path $readme_path"
+  echo "It is recommended to have a dedicated README file for each Docker image"
+  exit 1
```

Comment:
this may kill parent script, since we use `set -ex`

Reply:
I agree!

So what I'm thinking of doing is that I can remove "exit 1" and instead of that 

`echo "Continue script execution..."`
`readme_path=../$(realpath $(dirname $0)/README.md)`

I think 

`readme_path = ../README.md` would also work fine

Is there any better way of approaching this problem?

Thank You!

Reply:
what if you just call `exit 0`? The caller will treat it as successful exit.